solutions/deploy/cpd-image-build/scripts/image-build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+BUILD_OUTPUT=$(curl -X POST "https://api.${REGION}.codeengine.cloud.ibm.com/v2/projects/${PROJECT_ID}/build_runs" -H "Authorization: ${TOKEN}" -H "Content-Type: application/json" -d '{ "build_name": "cpd-build" }')
```

Comment:
Is it not possible to use the provider? https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/code_engine_build

Reply:
I do create the project/build/etc through the code engine DA.  The problem is executing a build run and checking for completion which there is no terraform for